.travis.yml

```diff
@@ -16,12 +19,11 @@ addons:
     - python3-pip
 
 before_install:
-  - sudo pip3 install --upgrade pip setuptools
-  - sudo pip3 install --ignore-installed six
-  - sudo pip3 install fusesoc
+  -  docker pull librecores/librecores-ci-dev
```

Comment:
You are using wrong image. https://hub.docker.com/r/librecores/librecores-ci/ should be used